addons/project/static/src/components/project_task_name_with_subtask_count_char_field/project_task_name_with_subtask_count_char_field.xml

```diff
@@ -2,6 +2,10 @@
 <templates xml:space="preserve">
 
     <t t-name="project.ProjectTaskNameWithSubtaskCountCharField" t-inherit="web.CharField" t-inherit-mode="primary" owl="1">
+        <xpath expr="//span[@t-esc='formattedValue']" position="replace">
+            <s t-if="['1_done', '1_canceled'].includes(this.props.record.data.state) and ['kanban', 'list'].includes(this.props.record.activeFields[this.props.name].viewType)" t-esc="formattedValue"/>
+            <span t-else="" t-esc="formattedValue"/>
+        </xpath>
```

Comment:
Why not doing this:
```suggestion
        <xpath expr="//span[@t-esc='formattedValue']" position="before">
            <s t-if="['1_done', '1_canceled'].includes(this.props.record.data.state) and ['kanban', 'list'].includes(this.props.record.activeFields[this.props.name].viewType)" t-esc="formattedValue"/>
        </xpath>
        <xpath expr="//span[@t-esc='formattedValue']" position="attributes">
            <attribute name="t-else">""</attribute>
        </xpath>
```
? 🤔 

What is the `s` element? 🤔 
You don't want to put `span` instead?

Reply:
oh right 🙂

Reply:
> Why not doing this:
> 
> ? 🤔
> 
> What is the `s` element? 🤔 You didn't want to put `span` instead?

Don't forget to check that when you have time, I think we can avoid doing a replace here. 🙂 

Thanks in advance!

Reply:
Yes sorry :) it's done

Reply:
cool 🤩